### DISCUSSION TITLE
Error linking latest nightly on Mac

### DISCUSSION THREAD

**Discussion by mooncalf** [nesbox/TIC-80#1853] [Q&A]
Hey there...

I'm getting this error on Mac when trying to build--

Undefined symbols for architecture x86_64:
  "_tic_project_load", referenced from:
      _cmdLoadCart in libtic80studio.a(console.c.o)
      _updateProject in libtic80studio.a(console.c.o)
      _onLoadCommandConfirmed in libtic80studio.a(console.c.o)
      _loadCover in libtic80studio.a(surf.c.o)
  "_tic_project_save", referenced from:
      _saveCartName in libtic80studio.a(console.c.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [bin/tic80] Error 1
make[1]: *** [CMakeFiles/tic80.dir/all] Error 2
make: *** [all] Error 2

Any help would be appreciated. I was able to build previous versions without issue, but I did upgrade to Monterey since then.

Thanks.

**Comment by nesbox** [comment]
I'll try to check when a Mac will be around.
Thanks.

**Reply by mooncalf** [reply]
Thanks! I need to trade this thing in.
